docs/tutorial.md

```diff
@@ -0,0 +1,521 @@
+
+# Hello World Task
+
+The main objective of the Pipeline CRD is to run your task individualy or in a pipeline. Every task runs as a Pod on your Kubernetes cluster with each step as a its own container.
```

Comment:
suggestion:
`The main objective of the Pipeline CRD is to run your task individually or as part of a pipeline`